src/poetry/console/commands/build.py

```diff
@@ -11,7 +11,8 @@ class BuildCommand(EnvCommand):
     description = "Builds a package, as a tarball and a wheel by default."
 
     options = [
-        option("format", "f", "Limit the format to either sdist or wheel.", flag=False)
+        option("format", "f", "Limit the format to either sdist or wheel.", flag=False),
+        option("output", "o", "Set output directory name", default="dist"),
```

Comment:
```suggestion
        option("output", "o", "Set output directory name", flag=False, default="dist"),
```

This should fix the pipeline :)

Reply:
Thanks!